fhirResourcesToLoad/1. practitioner.json

```diff
@@ -64,6 +64,32 @@
       "issuer": {
         "display": "Example University"
       }
+    },
+    {
+      "identifier": [
+        {
+          "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+          "value": "2183126"
+        }
+      ],
+      "code": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/v2-0360/2.7",
```

Comment:
I spent some time searching and I don't think there's any existing codesystem for this. We discussed offline and it will probably make sense to profile `Practitioner` and/or make our own code system.